source/unified-test-format/unified-test-format.rst

```diff
@@ -3,13 +3,13 @@ Unified Test Format
 ===================
 
 :Spec Title: Unified Test Format
-:Spec Version: 1.5.6
+:Spec Version: 1.5.7
```

Comment:
this seems like it could be minor too, but I am not totally sure on how spec versions here change vs  schema versions so I will defer to @jmikola

Reply:
This should definitely have been 1.6.0, since new schema changes were introduced.

Reply:
Yeah changed it to `1.6.0` with the "Use minor versions" commit.